### DISCUSSION TITLE
MasterBattery Upower UE-48Li100-3U

### DISCUSSION THREAD

**Discussion by emuman049** [Louisvdw/dbus-serialbattery#567] [New BMS requests]
Uses RS485 for battery to battery communication and to external via ES485, CAN bus or RS232.
Supported inverters and protocols:
MasterPower
Voltronic Protocol 2020.07/09
Schneider Version2 SE
Growatt RS485 protocol ESS rev 2.01
Growatt CAN Bus Low voltage V1.04
SRNE Studer BMS protocol V1.02_EN
GOODWE 
KSTAR
EAST
SMA

**Comment by Louisvdw** [comment]
Is the protocol document for the BMS available?

**Comment by drmiar** [comment]
What about Victron inverters? 
Which PC- software can bei used in order to list all values and to update the firmware?
